### PR TITLE
Allen/back button

### PR DESCRIPTION
**Pull request by algtan**
Added a `Back` button to the snack bar app:

1. Created a `Buttons` component for the button component with styling to match design
2. Created a `CodeScreen` component for when a user selects a currency (contains selection and the `Back` button)
3. Added `CodeScreen` to `MainScene`, where the screen change is from conditional rendering

### CODE REVIEW

File: src/components/CodeScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+export function CodeScreen(props: CodeScreenProps): JSX.Element {
+  const handleBackBtnClick = (): void => {
+    props.setShowCodeScreen(false)
+  }
+
+  return (
+    <>
+      <div style={codeScreenStyle}>
+        <div style={textStyle}>YOU'VE SELECTED {props.coinSelection}</div>
```
Comment (thehobbit85) [inline src/components/CodeScreen.tsx:27]: Avoid 'literal strings'.
Create a constant json file and import from there.

Comment (thehobbit85) [inline src/components/CodeScreen.tsx:27]: Avoid 'literal strings'.
Create a constant json file and import from there.

File: src/MainScene.tsx [hunk 2]
```diff
@@ -63,17 +66,34 @@ export function MainScene(): JSX.Element {
...
+            .sort((a, b) => a.localeCompare(b))
+            .map(option => (
+              <React.Fragment key={option}>
+                <button onClick={() => handleOptionClick(option)}>
+                  {option}: {usdToCoinRates[option]}
+                </button>
+                <br />
+              </React.Fragment>
+            ))}
+        </>
```
Comment (thehobbit85) [inline src/MainScene.tsx:95]: This part can be separated into its own component.

**Review by thehobbit85 — CHANGES_REQUESTED** [review]
See comments.

File: src/components/CodeScreen.tsx (hunk 1 above)
Comment (algtan) [reply src/components/CodeScreen.tsx]: Updated per comment. A `constant.json` file was created in PR https://github.com/EdgeApp/edge-snack-bar-app/pull/2.

File: src/MainScene.tsx (hunk 2 above)
Comment (algtan) [reply src/MainScene.tsx]: This will be addressed in a future PR: https://github.com/EdgeApp/edge-snack-bar-app/pull/6.

**Review by thehobbit85 — APPROVED** [review]

**Pull request merged**